docs/kpod-logs.1.md

```diff
@@ -0,0 +1,44 @@
+% kpod(1) kpod-logs - Fetch the logs of a container
+% Ryan Cole
+# kpod-logs "1" "March 2017" "kpod"
+
+## NAME
+kpod logs - Fetch the logs of a container
+
+## SYNOPSIS
+**kpod** **logs** [*options* [...]] container
+
+## DESCRIPTION
+The kpod logs command batch-retrieves whatever logs are present for a container at the time of execution.  This does not guarantee execution order when combined with kpod run (i.e. your run may not have generated any logs at the time you execute kpod logs
+
+## OPTIONS
+
+**--details**
```

Comment:
I forgot that this needed to be hidden until the `--log-opt` flag could be implemented in `kpod create`.  I'll remove it from the man page for now